src/procstat.c

```diff
@@ -33,21 +33,26 @@
 
 
 #define FUSE_USE_VERSION 26
+
+#include "list.h"
+#include "procstat.h"
+
 #include <fuse/fuse_lowlevel.h>
-#include <stdbool.h>
+
+#include <sys/param.h>
+
+#include <assert.h>
+#include <ctype.h>
 #include <errno.h>
-#include "procstat.h"
-#include "list.h"
 #include <errno.h>
 #include <limits.h>
 #include <math.h>
-#include <stdlib.h>
 #include <pthread.h>
-#include <string.h>
-#include <assert.h>
-#include <sys/param.h>
-#include <ctype.h>
+#include <stdatomic.h>
```

Comment:
Are all these changes to the include's required here?

Reply:
It was just a reorganization. Not worthy of its own commit, but annoying enough not to ignore anymore.